ion-hash/tests/ion_hash_tests.rs

```diff
@@ -75,10 +100,16 @@ fn test_case<E: Element>(ion: &E, strukt: &E) -> IonResult<()> {
 
     let digest = TestDigest::default();
     let hasher = IonHasher::new(digest.clone());
-    let _result = hasher.hash_element(ion)?;
-
-    let (recorded_updates, recorded_digest) = digest.inspect();
-    let mut recorded_updates = recorded_updates.into_iter();
+    let annotations: Vec<_> = ion
+        .annotations()
+        .map(|it| it.text().unwrap().to_string())
+        .collect();
+    let test_case_name = match &annotations[..] {
+        [] => format!("unannotated-test-case #{}", case_number), // FIXME: Use Dumper to print the case name
+        [single] => single.clone(),
+        _ => unimplemented!(),
+    };
```

Comment:
Is this literally to get the test case name?  I would comment this intent, I stared at it for a bit wondering what it was doing.

Reply:
Yes it is. I'll extract a method.